core/arch/arm/pta/benchmark.c

```diff
@@ -202,7 +225,7 @@ void bm_timestamp(void)
 
 	cpu_buf = &bench_ts_global->cpu_buf[cur_cpu];
 	ts_i = cpu_buf->head++;
-	ts_data.cnt = read_pmu_ccnt() * TEE_BENCH_DIVIDER;
+	ts_data.cnt = read_cntpct();
```

Comment:
I suppose that should be `read_cntvct()` (which needs to be added).

Reply:
@jforissier 
CNTVCT_EL0 should be used in case if we are running under hypervisor (it's equal to the physical count value visible in CNTPCT_EL0 minus the virtual offset visible in CNTVOFF_EL2), But, as we are running on S-EL1, and obviously S-EL2 doesn't exist, I guess CNTVCT_EL0 == CNTPCT_EL0 here.

I've added `timer_enable_el0_access` to be able to read CNTVCT_EL0/CNTPCT_EL0 from NW side, maybe it adds a bit of confusion here.

There is another concert:
Should I read CNTVCT_EL0 in NW in case if ARMv8.1-VHE is implemented, or it's better to stick to CNTPCT_EL0